config/default.yaml

```diff
@@ -33,7 +33,8 @@ appInsights:
   roleName: "ccd-api-gateway"
 secrets:
   ccd:
-    ccd-api-gateway-oauth2-client-secret: ccd_gateway_secret
+    ccd-api-gateway-oauth2-client-secret: ""
     postcode-info-address-lookup-token: AA
-    microservicekey-ccd-gw: AAAAAAAAAAAAAAAC
-    AppInsightsInstrumentationKey: "some-key"
\ No newline at end of file
+    microservicekey-ccd-gw: ""
```

Comment:
Does the app still start up with this? the intent is that it fails to start not that it starts with an empty value.

You can work around test issues by setting NODE_ENV to be test and then loading dummy values there

Reply:
@nathan-clark can you have a look at @timja comment please